react-example/.env.example

```diff
@@ -8,3 +8,6 @@
 
 # Convenience variable to automatically set the login email during testing.
 # LOGIN_EMAIL=email@email.com
+
+# IS_EU_ITERABLE_SERVICE=false
+# DANGEROUSLY_ALLOW_JS_POPUP_EXECUTION=false
```

Comment:
Why do we need this at the sample app level?

Reply:
When the package is installed via package manager, devs don't have access to modify the `.env` file of the web-sdk so they need to be able to pass this value in from their own `.env`. This just serves as an example of how to do that.